example-templates/basic.html

```diff
@@ -10,20 +10,18 @@
   <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
   <script>/*global hljs*/ hljs.initHighlightingOnLoad();</script>
   <style>
-    #map {
-      height: 256px;
-      margin: 1em;
+    .userchip {
```

Comment:
Please rename to more generic class name or move to example code itself.

Reply:
remove style definition in template.